modules/tinymce/src/plugins/lists/test/ts/browser/ListPropertiesTest.ts

```diff
@@ -132,4 +134,52 @@ describe('browser.tinymce.plugins.lists.ListPropertiesTest', () => {
     await TinyUiActions.pWaitForUi(editor, '.tox-collection__item:contains("List properties"):not(.tox-collection__item--state-disabled)');
     TinyUiActions.clickOnMenu(editor, '.tox-mbtn:contains("Custom")');
   });
+
+  it('TINY-6907: List properties command is used to update the DOM', async () => {
```

Comment:
This test title initially confused me, if I'm understanding correctly, maybe something like:

```suggestion
  it('TINY-6907: List properties dialog uses mceListUpdate command internally to update the DOM', async () => {
```